fawltydeps/settings.py

```diff
@@ -100,21 +100,42 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
 
 DEFAULT_IGNORE_UNUSED = {
     # Development tools not meant to be imported
+    # Formatting Tools
+    "autopep8",
     "black",
+    "isort",
+    "pyformat",
+    "yapf",
+    # Linting Tools
+    "bandit",
+    "flake8",
+    "mccabe",
+    "pyflakes",
+    "pylint",
+    # Security Tools
     "codespell",
-    "colorama",
+    # Documentation Tools
+    "sphinx",
+    "sphinx-rtd-theme",
+    "recommonmark",
+    # Testing Tools
     "coverage",
     "fawltydeps",
-    "flake8",
     "hypothesis",
-    "isort",
     "mypy",
     "nox",
-    "pylint",
-    "pyright",
+    "pre-commit",
     "pytest",
-    "wheel",
     "tox",
+    # Utility Tools
+    "colorama",
+    "pydocstyle",
+    "pyright",
+    "rope",
+    "ruff",
+    "twine",
+    "unify",
+    "wheel",
```

Comment:
- I'd put `pyright` and `mypy` under type-checking or linting.
- `ruff` is a linter
- `unify` should go under formatting
- `twine` and `wheel` under building and packaging

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/939043f6fe59516cda60b0d13227d77f50fb114b.